scss/main.scss

```diff
@@ -391,6 +389,28 @@ body {
 			align-items: flex-start;
 		}
 	}
+	&__button {
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		margin: auto;
+		margin-top: 50px;
+		padding: 10px 20px;
+		background-color: #e68ae3;
```

Comment:
źle to ująłem chyba z tym kolorem 😉 chciałem się tego oczojebnego różu pozbyć.

Chodziło mi by kolor przycisku był: #94555c, font: #ffffff, border-color: #94555c

ale chyba masz rację, lepiej będzie jak font będzie: #94555c, broder-color: #94555c, tło przycisku przezroczyszte. A hover dać odwrócony (tło w kolorze i font biały)? będzie spójniejsze z tymi przyciskami u góry? jak myślisz?